docs/static/mystyles.css

```
@@ -3,6 +3,51 @@ ul {
  padding-left: 2.5em !important;
 }
 
+:root {
+  --table-head: #e6e6e6;
+  --table-border: #bbb;
+  --table-row-even: #f9f9f9;
+  --table-row-odd: #f0f0f0;
+}
+
+:root[data-theme="dark"] {
+  --table-head: #242424;
+  --table-border: #444;
+  --table-row-even: #121212;
+  --table-row-odd: #1a1a1a;
+}
+
+h2 {
+  font-size: 1.5em !important;
```

What do you think about a bit of padding? The empty lines between the description and the heading don't seem to be translated to whitespace in the HTML.
```suggestion
h2 {
  padding-top: 0.5em !important;
  font-size: 1.5em !important;
```

---

Great idea. :+1: Implemented your suggestion.